apps/web/src/app/(dashboard)/admin/documents/data-table-action-button.tsx

```diff
@@ -0,0 +1,30 @@
+'use client';
+
+import Link from 'next/link';
+
+import { Edit } from 'lucide-react';
+import { useSession } from 'next-auth/react';
+
+import { Document } from '@documenso/prisma/client';
+import { Button } from '@documenso/ui/primitives/button';
+
+export type DataTableActionButtonProps = {
+  row: Pick<Document, 'id'>;
+};
+
+export const DataTableActionButton = ({ row }: DataTableActionButtonProps) => {
+  const { data: session } = useSession();
+
+  if (!session) {
+    return null;
+  }
```

Comment:
This seems redundant no?

Reply:
🤦‍♂️ I thought I removed the component altogether since it only renders the button. Thanks for the catch!